src/uct/api/uct_def.h

```diff
@@ -398,7 +407,14 @@ typedef struct uct_cm_ep_server_conn_notify_args {
     uint64_t                   field_mask;
 
     /**
-     * Indicates the client's @ref ucs_status_t status.
+     * Indicates the client's @ref ucs_status_t status:
+     * UCS_OK                   - the client completed its connection
+     *                            establishment successfully.
```

Comment:
`the client completed its connection establishment and called @ref uct_cm_client_ep_conn_notify`